Sources/Helpers/OAAppSettings.m

```diff
@@ -32,14 +32,15 @@
 #define metricSystemChangedManuallyKey @"metricSystemChangedManuallyKey"
 #define liveUpdatesPurchasedKey @"liveUpdatesPurchasedKey"
 #define settingOsmAndLiveEnabledKey @"settingOsmAndLiveEnabledKey"
+#define liveUpdatesRetryesKey @"liveUpdatesRetryesKey"
```

Comment:
Did you mean retries?

Reply:
yes, but the setting storage key will remain "retryes", as it is written in android